resolvers/query.js

```diff
@@ -110,5 +137,36 @@ module.exports = {
     },
     user(_, { id }) {
         return User.findById(id);
+    },
+    async timeSheetRecordsToCSV(_, { userId, projectId, startDate, endDate }) {
+        const records = await TimesheetRecord.findAll({
+            raw: true,
+            where: {
+                userId: userId,
+                projectId: projectId,
+                date: {
+                    [Op.between]: [startDate, endDate]
+                }
+            },
+            include: [
+                {
+                    model: Project,
+                    as: 'project',
+                    attributes: ['name'],
+                    where: {
+                        id: projectId
+                    }
+                },
+                {
+                    model: User,
+                    as: 'user',
+                    attributes: ['name'],
+                    where: {
+                        id: userId
+                    }
+                }
+            ]
+        });
+        return toCSVAndUploadToS3(await preferToCVS(records));
```

Comment:
Why is there an `await` here? preferToCVS doesn't seem to be asynchronous

Reply:
fix